iframe/priv/static/iframe/v3.html

```diff
@@ -80,6 +81,11 @@
           handleSync(callback) {
             state.syncHandler = callback;
           },
+
+          requestSecret(currentSecret, callback) {
+            state.secretHandler = callback;
+            postMessage({ type: "secrets", currentSecret });
```

Comment:
To make the flow clearer, let's make the messages "select secret" when going all the way to the LV, and then "secret selected" once selected/created (instead of "update secret").

```suggestion
            postMessage({ type: "selectSecret", currentSecret });
```